### DISCUSSION TITLE
Is it possible to launch Sandboxie driver and Service after Windows boots?

### DISCUSSION THREAD

**Discussion by HqT65W6R** [sandboxie-plus/Sandboxie#4326] [Q&A Feedback]
I can only launch Sbie driver and service after Windows already boots. If i want to disable it, then I have to do so and reboot for that to take effect. Is it possible to avoid rebooting?

**Comment by offhub** [comment]
To stop the Sandboxie service and driver, you can navigate to:

    Sandboxie Plus > Sandbox > Maintenance > Stop All.

This will halt all running sandboxes and stop the associated services and drivers.

To start it again, simply running Sandboxie Plus will automatically start the necessary services and drivers.
